controllers/postComment.controller.js

```diff
@@ -0,0 +1,16 @@
+const { checkArticleExist } = require("../models/checkArticle.model");
+const { insertComment } = require("../models/insertComment.model");
+
+exports.postComment = (req, res, next) => {
+  const { article_id } = req.params;
+  const { username, body } = req.body;
+
+  const promises = [checkArticleExist(article_id),insertComment(article_id, username, body), ];
```

Comment:
We don't need to check if the article exists first, sql will handle this error if the article id violates foreign key restraint on the post. I'd suggest refactoring this to be cleaner.